master/Puppetfile

```diff
@@ -0,0 +1,9 @@
+forge "https://forgeapi.puppetlabs.com"
```

Comment:
I don't know the implications that's what the example looks like: https://github.com/rodjek/librarian-puppet